### DISCUSSION TITLE
Problems during installation

### DISCUSSION THREAD

**Discussion by mpaglia0** [mosparo/mosparo#171] [Problems & Issues] [answered]
Mosparo is installed and Javascripts seems to work but my mosparo box shown an error
(An error occourred. Please try again).
If i try to debug the problem with the browser console I see token cannot be loaded because the script calls
www.mauriziopaglia.it/www.mosparo.mauriziopaglia.it (error 404).
If you desire to see it live this is the link:
https://www.mauriziopaglia.it/contattami.html

Thanks,
Maurizio

**Comment by zepich** [comment, marked as answer]
Hi @mpaglia0 

Thank you very much for your question.

The protocol needs to be included. In the JavaScript file (https://www.mauriziopaglia.it/theme/js/mosparo.js), you have the URL `www.mosparo.mauriziopaglia.it` without the protocol (`https://`). The browser will use it as a path instead of a correct URL. 

If you add the `https://`, everything should work.

```javascript
var m;
window.onload = function(){
	 m = new mosparo(
		 'mosparo_spambox',
		 'https://www.mosparo.mauriziopaglia.it', // Add https://
		 'f37134f6-7b19-4a25-bac9-0783110aaa85',
		 'lwEKO4E54VkmFpV5K-y8mSCRGVPAAcmB2C5Uuy-XZMU'
	 );
};
```

Let me know if you have any other questions.

Kind regards,

zepich

**Reply by mpaglia0** [reply]
IT FINALLY WORKS!
Thank you indeed.
Now I need to refine my CSP, test Mosparo, rest and wish you a happy new year!!

**Reply by zepich** [reply]
I'm happy to hear that!

Thank you very much! I wish you the same!
